cves/CVE-2014-9604.yml

```diff
@@ -64,7 +64,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: Due to a lack of checking for a zero value of a slice, a remote attacker could cause an out-of-bounds array access error which could potentially deny service, or other possible greater impacts than just denial of service.
```

Comment:
Possibly split into multiple lines? This actually goes over the edge of the github preview. It shouldn't matter a huge amount but it's inconsistent with the rest of the formatting.

Reply:
Agreed, I personally prefer doing the | and then newline text, it makes it a lot more readable and nicer to look at.

Reply:
I also feel like the end of the sentence is awkwardly structured. The "...which could potentially deny service, or other possible greater impacts than just denial of service." You can probably rephrase to "...which could potentially deny service or worse." Or a more specific example for the 2nd part of that